terraform/variables.tf

```
@@ -86,5 +86,5 @@ variable "base_domain" {
 
 variable "trade_tariff_oauth_secret" {
   description = "trade tarrif oauth secrets."
-  type        = "map"
+  type        = map(string)
```

```suggestion
  type        = string
```

I think this should just be a `string` here, because this is the value of `TRADE_TARIFF_OAUTH_SECRET`